ravenframework/utils/InputData.py

```diff
@@ -534,6 +534,9 @@ def returnDefault(self, name):
     sub = self.getSub(name)
     if sub is not None:
       return sub.default
+    else:
+      pass #uncomment next line if needed for debugging.
+      #raise ValueError("Unrecognized variable '{}'".format(name))
```

Comment:
Should these lines stay in for debugging, or can the be removed?  (FYI: the docstring for the function says that this returns None, not raise an error for subs that are not found.)

Reply:
@Jimmy-INL Could you provide your answer for Josh's comment here?

Reply:
These lines were actually added by @joshua-cogliati-inl. I have no issues with removing them. @joshua-cogliati-inl, please confirm that we do not need them and I will remove them.

Reply:
Yes, we can remove them.